gui/src/components/widgets/SkeletonVisualizerWidget.tsx

```diff
@@ -110,7 +110,7 @@ export function ToggleableSkeletonVisualizerWidget(
         </Button>
       )}
       {enabled && (
-        <>
+        <div className="flex flex-col gap-2">
```

Comment:
why?

Reply:
small gap between button and background of 3D preview. Without gap it renders together and it does not fit actual layout:
![image](https://github.com/user-attachments/assets/ab0b092c-dbff-4583-b134-78b14ce3eacb)
with gap:
![image](https://github.com/user-attachments/assets/ad27a677-5bb0-45d6-8614-b7caeae25a10)